src/services/TestResultsService.ts

```diff
@@ -1,16 +1,12 @@
-import { ValidationResult, string, any, validate } from "joi";
 import moment from "moment-timezone";
-import { cloneDeep, mergeWith, isEqual, differenceWith } from "lodash";
+import { isEqual, differenceWith } from "lodash";
 import * as enums from "../assets/Enums";
 import * as models from "../models";
-import * as validators from "../models/validators";
-import * as utils from "../utils";
 import { VehicleTestController } from "../handlers/VehicleTestController";
 import { TestDataProvider } from "../handlers/expiry/providers/TestDataProvider";
 import { DateProvider } from "../handlers/expiry/providers/DateProvider";
-import { MappingUtil } from "../utils/mappingUtil";
-import { ValidationUtil } from "../utils/validationUtil";
-import { TestTypeParams } from "../models/TestTypeParams";
+import { createVerify } from "crypto";
```

Comment:
seems like some autofixing got wrong? :)
We don't seem to be using this

Reply:
removed.